be/src/testing/testing.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { TestingService } from './testing.service';
+import { TestingResolver } from './testing.resolver';
+import { ItemModule } from 'src/item/item.module';
+import { CategoryModule } from 'src/category/category.module';
+import { NoteModule } from 'src/note/note.module';
+import { UserModule } from 'src/user/user.module';
+
+@Module({
+  imports: [ItemModule, CategoryModule, NoteModule, UserModule],
+  providers: [TestingResolver, TestingService],
+  exports: [TestingService],
```

Comment:
why is this exported?

Reply:
removed in [0095b0](https://github.com/TPT-Loane/TPT-Loane/pull/120/commits/0095b0db85252a7760b0493ae5789556f1ba628e)